docling/pipeline/standard_pdf_pipeline.py

```diff
@@ -600,7 +612,7 @@ def _integrate_results(
         ]
         if proc.is_complete_failure:
             conv_res.status = ConversionStatus.FAILURE
-        elif proc.is_partial_success:
+        elif timeout_exceeded or proc.is_partial_success:
```

Comment:
@cau-git Could we not have a specific ConversionStatus value for this, eg `ConversionStatus.TIME_OUT`